packages/react-components/src/button/src/IconButton.css

```diff
@@ -62,6 +62,12 @@
 
 /* INHERIT */
 .o-ui-button-inherit.o-ui-button-solid .o-ui-button-icon {
-    fill: var(--white);
+    fill: var(--static-white);
     opacity: .9;
 }
+
+.o-ui-icon-button .o-ui-button-icon,
```

Comment:
Not sure about this selector? It seem related to this?

![image](https://user-images.githubusercontent.com/794579/104750923-b6a98680-5722-11eb-811d-787c65c77485.png)